doc/protocol.md

```diff
@@ -26,6 +26,42 @@ If an outgoing connection to a non-verified peer is accepted and an `ExchangePub
 If a protocol violation is detected at any moment during communication (allowing to assume the remote end runs something else, not Qubic node), then the IP is removed even if it is verified.
 An IP is only removed from the list of peers if the list still has at least 10 entries afterwards and if it is not in the initial `knownPublicPeers`.
 
+## Broadcast Message
+
+Defined in https://github.com/qubic/core/blob/main/src/network_messages/broadcast_message.h
+
+```
+struct BroadcastMessage
+{
+    m256i sourcePublicKey;
+    m256i destinationPublicKey;
+    m256i gammingNonce;
+
+    enum {
+        type = 1,
+    };
+};
+```
+
+- SourcePublicKey must not be NULL.
+- The message signature must be verified with SourcePublicKey.
+- SourcePublicKey's balance must be greater than MESSAGE_DISSEMINATION_THRESHOLD or SourcePublicKey is a computor's public key, and the message is broadcasted.

```

Comment:
I would write "The message is broadcasted if sourcePublicKey's balance is greater than MESSAGE_DISSEMINATION_THRESHOLD or sourcePublicKey is a computor's public key"